linux_os/guide/services/ntp/ensure_chrony_is_configured/oval/shared.xml

```diff
@@ -0,0 +1,20 @@
+<def-group>
+  <definition class="compliance" id="{{{ rule_id }}}" version="1">
+    {{{ oval_metadata("A remote NTP Server for time synchronization should be specified (and dependencies are met)") }}}
+    <criteria comment="chrony.conf conditions are met">
+      <criterion test_ref="test_chrony_remote_server_configuration" />
+    </criteria>
+  </definition>
+
+  <ind:textfilecontent54_test check="all" check_existence="at_least_one_exists"
+  comment="Ensure at least one remote NTP server is set" id="test_chrony_remote_server_configuration" version="1">
+    <ind:object object_ref="object_chrony_remote_server_configuartion" />
+  </ind:textfilecontent54_test>
+
+  <ind:textfilecontent54_object comment="Ensure at least one remote NTP server is set"
+  id="object_chrony_remote_server_configuartion" version="1">
+    <ind:filepath operation="pattern match">^/etc/chrony.conf$</ind:filepath>
```

Comment:
I believe here you might want to use: {{{ chrony_conf_path }}}
see other rules that have this variable use